src/containers/Login/index.tsx

```diff
@@ -133,9 +133,21 @@ const Login = ({ toggleSnackbar, setSnackbarMessage, setLoading }: ILoginProps)
 
   return (
     <>
-      <img src={polygon1} alt='' style={{ position: 'absolute', top: '3%', left: '-10%' }} />
-      <img src={polygon2} alt='' style={{ position: 'absolute', top: '50%', right: '20%' }} />
-      <img src={polygon3} alt='' style={{ position: 'absolute', right: '20%', top: '5%', width: '200px' }} />
+      <CardMedia
```

Comment:
Instead of writing duplicate code, I could move this to a different component and then import it into Login/Register.